internal/tailer/logstream/decode.go

```diff
@@ -22,6 +22,10 @@ func decodeAndSend(ctx context.Context, lines chan<- *logline.LogLine, pathname
 		rune  rune
 		width int
 	)
+	if partial == nil {
```

Comment:
The assumption is that a UDP datagram is a single message, and that you will get the whole message on a read, there is never a partial, and it is especially not delimited by a new line.  Perhaps this should get pushed into the UDP message loop, rather than extending behaviour here?

Reply:
Aha I see, thanks.